code/modules/cm_preds/yaut_items.dm

```diff
@@ -1164,6 +1162,69 @@
 				new_access = list(ACCESS_YAUTJA_SECURE, ACCESS_YAUTJA_ELITE, ACCESS_YAUTJA_ELDER, ACCESS_YAUTJA_ANCIENT)
 	access = new_access
 
+/obj/item/tool/yautja_cleaner
+	name = "cleanser gel vial"
+	desc = "A small vial containing a liquid capable of dissolving the gear of the fallen whilst in the field."
+	icon = 'icons/obj/items/hunter/pred_gear.dmi'
+	icon_state = "dissolving_vial"
+	force = 0
+	throwforce = 1
+	w_class = SIZE_SMALL
+	flags_item = ITEM_PREDATOR
+	black_market_value = 150
+
+	var/image/dissolving_image
+
+/obj/item/tool/yautja_cleaner/afterattack(obj/item/target, mob/user, proximity)
+	if(!isitem(target))
+		return
+	if(loc != user) //Early returns if the cleaner has been inserted into a container. Whether or not this happens is based on the user's intent; see storage.dm for info.
+		return
+	if(!can_dissolve(target, user))
+		return
+	handle_dissolve(target, user)
+
+/obj/item/tool/yautja_cleaner/proc/can_dissolve(obj/item/target, mob/user)
+	if(!HAS_TRAIT(user, TRAIT_YAUTJA_TECH))
+		to_chat(user, SPAN_WARNING("You have no idea what this even does."))
+		return FALSE
+	if(HAS_TRAIT(target, TRAIT_ITEM_DISSOLVING))
+		to_chat(user, SPAN_WARNING("\The [target] is already covered in dissolving gel."))
+		return FALSE
+	if(user.alpha < 255)
+		to_chat(user, SPAN_WARNING("It would not be safe to attempt this while cloaked!"))
+		return FALSE
+	if(target.anchored)
+		to_chat(user, SPAN_WARNING("\The [target] cannot be moved by any means, why dissolve it?"))
+		return FALSE
+	if(istype(target.loc, /mob/living))
+		to_chat(user, SPAN_WARNING("You cannot dissolve \the [target] while it is being held."))
+		return
+	if(istype(target, /obj/item/tool/yautja_cleaner))
```

Comment:
why not

Reply:
Another player picking up the item should interrupt it. I suppose it could be changed to check for if it's being held by anything other than the user of the dissolving gel, so that you can dissolve items in *your* hands.